presto-main/src/test/java/com/facebook/presto/cost/TestComparisonStatsCalculator.java

```diff
@@ -178,6 +178,11 @@ public void setUp()
         };
     }
 
+    private SymbolStatsEstimate updateNDV(SymbolStatsEstimate symbolStats, double delta)
+    {
+        return symbolStats.mapDistinctValuesCount(ndv -> ndv + delta);
```

Comment:
`min(0, ...)`

Reply:
why?

Reply:
to make sure that ndv is not less than zero

Reply:
This constructs an expected value in a test. If test expects -1, this is the test's responsibility. Currently we don't have generic tests that would generate random conditions and would need this logic.